tests/test_e2e_10_mef_eline.py

```diff
@@ -719,11 +719,97 @@ def evc_intra_switch_without_VLAN_tag(self):
 
     def create_many_EVC_at_once_and_verify_proper_installtion(self):
         # TODO Create many EVC at once and check if they are all working (e.g., 300 EVCs in the same file)
-        #  check if vlan-id is inside the dump-flows
-        assert True
+
+        url = "http://67.17.206.252:8181/api/kytos/mef_eline/v2/evc/"
```

Comment:
I think this test should be in another PR, right? It also contains a hard-coded IP that should be modified.